.gitignore

```diff
@@ -25,6 +25,8 @@ hs_err_pid*
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+.idea
```

Comment:
please make a copy of your idea folder, delete it, push the changes and then restore it

Reply:
Ready